Duvet/Configuration/SheetConfiguration.swift

```diff
@@ -25,6 +25,11 @@ public struct SheetConfiguration: Equatable {
     /// The number of points that the sheet should sit below the top safe area.
     public var topInset: CGFloat
 
+    /// A flag indicating if this sheet should use Duvet's default keyboard avoidance or not. If `true`,
+    /// Duvet will handle keyboard avoidance, otherwise it will do nothing when the keyboard is presented
+    /// or dismissed.
+    public var usesDefaultKeyboardAvoidance: Bool
```

Comment:
I like "avoidance"! I haven't heard that before. I really like all the names out there for this. I've heard it called keyboard dodging too, which I very much enjoy the visual of.

Reply:
Haha dodging is a great term for it!

Reply:
If you dodge a wrench, you can dodge a keyboard!